addons/purchase/report/purchase_report.py

```diff
@@ -119,6 +119,12 @@ def _from(self):
             currency_table=self.env['res.currency']._get_query_currency_table({'multi_company': True, 'date': {'date_to': fields.Date.today()}}),
         )
         return from_str
+    
+    def _where(self):
+        where_str = """
+            WHERE
+                l.display_type IS NULL"""
+        return where_str
```

Comment:
```suggestion
        return """
            WHERE
                l.display_type IS NULL
        """
```
Small details, otherwise lgtml

Reply:
fixed it :)